File: c#/program53.cs
using System;

class MainClass
{
    static bool CheckBit(int No)
    {
        int iMask = 0X00000108;
        long result = 0;

        result = iMask & No;
        if(result == iMask)
        {
            return true;
        }
        else
        {
            return false;
        }
    }

    static void Main()
    {
        int value = 0;

        Console.WriteLine("Enter number :");
        value = int.Parse(Console.ReadLine());

        bool bret =  CheckBit(value);
        if(bret == true)
        {
            Console.WriteLine("4th and 9th bit is ON");
        }
        else
        {
            Console.WriteLine("4th and 9th Bit is OFF :");
        }
    }
}

File: java/program53.java
import java.util.*;

class Bit
{
    public int No;

    Bit(int no)
    {
        this.No = no;
    }

    public boolean CheckBit()
    {
        int iMask = 0x00000004;
        int result = 0;

        result = iMask & No;
        System.out.println(iMask & No);
        if(result == iMask)
        {
            return true;
        }
        else
        {
            return false;
        }
    }
}

class program53
{
    public static void main(String arg[])
    {
        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter number");
        int value = sobj.nextInt();

        Bit obj = new Bit(value);
        boolean bRet = obj.CheckBit();

        if(bRet == true)
        {
            System.out.println("Bits are On");
        }
        else
        {
            System.out.println("Bits are Off");
        }
    }
}